dashboard/lib/services/curriculum_pdfs/resources.rb

```diff
@@ -222,8 +223,9 @@ def fetch_url_to_path(url, path)
           )
 
           ChatClient.log(
-            "<@teacher-tools-on-call> Please follow instructions in https://docs.google.com/document/d/1mBY56DeAzrwTM3CVIOFho3azTi9mudE37ZQrVZXxaMA/edit#heading=h.axfu5or8cueg to troubleshoot",
-            color: 'yellow'
+            "Please follow instructions in https://docs.google.com/document/d/1mBY56DeAzrwTM3CVIOFho3azTi9mudE37ZQrVZXxaMA/edit#heading=h.axfu5or8cueg to troubleshoot",
+            color: 'yellow',
+            notify_group: 'teacher-tools-on-call'
```

Comment:
Any further adoption will need update in the chat client mapping as well. This isn't very obvious now, and will be detected only when testing by seeing the messages emitted not actually tagging the group.